public/app/features/dashboard-scene/serialization/transformSaveModelToScene.ts

```diff
@@ -67,7 +68,10 @@ export function transformSaveModelToScene(rsp: DashboardDTO): DashboardScene {
     autoMigrateOldPanels: false,
   });
 
-  return createDashboardSceneFromDashboardModel(oldModel);
+  const scene = createDashboardSceneFromDashboardModel(oldModel);
+  scene.state.$behaviors?.push(registerDashboardSceneTracking(oldModel, rsp.dashboard.version));
```

Comment:
What's the reason this behavior is added here and not inside createDashboardSceneFromDashboardModel?